utoipa-swagger-ui/Cargo.toml

```diff
@@ -34,3 +34,6 @@ rustdoc-args = ["--cfg", "doc_cfg"]
 [build-dependencies]
 zip = { version = "0.6", default-features = false, features = ["deflate"] }
 regex = "1.7"
+reqwest = { version = "0.11", features = ["blocking"] }
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Is the openssl needed? I don't see any usecases?

Reply:
I think reqwest requires openssl (reqwest -> native-tls -> openssl) but then it should pull it so no need to add it to Cargo.tom. I removed it . Cf. commit.